docs/System_toolchain.rst

```diff
@@ -0,0 +1,148 @@
+.. _system_toolchain:
+
+``system`` toolchain
+====================
+
+In EasyBuild v4.0, the infamous ``dummy`` toolchain was deprecated, and replaced with the ``system`` toolchain.
```

Comment:
"was deprecated" -> "has been deprecated" or is it "have been"?

Reply:
`has been` works for me (`have` is plural, no?)